cns/NetworkContainerContract.go

```diff
@@ -131,6 +131,7 @@ type IPConfiguration struct {
 // SecondaryIPConfig contains IP info of SecondaryIP
 type SecondaryIPConfig struct {
 	IPAddress string
+	NCVersion int
```

Comment:
Can you add a comment why you are persisting NCVersion with SecondaryIP

Reply:
Sure.